src/base/materialized.ts

```diff
@@ -230,60 +236,203 @@ export class MaterializedChatMessage implements IMaterializedNode {
 		return tokenizer.countMessageTokens({ ...this.toChatMessage(), content: '' });
 	});
 
-	private readonly _text = once(() => {
-		let result = '';
+	private readonly _text = once((): (string | MaterializedChatMesageImage)[] => {
+		let result: (string | MaterializedChatMesageImage)[] = [''];
```

Comment:
I think you could leave off the initial element and things would work fine.

```suggestion
		let result: (string | MaterializedChatMesageImage)[] = [];
```